apps/signing/tests/test_views.py

```diff
@@ -168,6 +172,50 @@ def test_version_exists(self):
         assert response.status_code == 200
         assert 'processed' in response.data
 
+    @mock.patch('devhub.tasks.Version.from_upload')
+    def test_version_exists_with_pk(self, version_from_upload):
+        # Mock Version.from_upload so the version won't be created.
+        self.create_version('3.0')
```

Comment:
Yeah using the view is definitely not ideal but there are a ton of edge cases around creating add-ons and versions and I wanted it to be as close to real as possible. The tests are definitely slow, unfortunately.